Clinical Key.js

```diff
@@ -130,6 +140,117 @@ function doWeb(doc, url){
 		search.setSearch({ ISBN: isbn });
 		search.getTranslators();
 	}
+	else if (contentType == 'journalArticle') {
+		eid = url.split('/');
+		pii = eid.pop().slice(7);
+		
+		/* can use xpath to get pii; guessing the url is faster
+		var eid = ZU.xpathText(doc, "//ul/@data-eid");
+		pii = eid.slice(7);*/
+		
+		if(!/^S(\d{15}X|\d{16})/.test(pii)){
+			throw new Error('PII incorrectly formatted');
+			return;
+		}
+		
+		doSearch(pii, doc);
+	}
+	
+}
+
+//Search & Processing based on CrossRef.js translator
+function doSearch(pii, doc){
+	crossRefQuery = 'http://api.crossref.org/works?query=' + pii
+	Zotero.Utilities.HTTP.doGet(crossRefQuery, function(responseText) {
+		processCrossRefREST(responseText, doc);
+		Zotero.done();
+	});
+	
+	Zotero.wait();
+}
+
+function processCrossRefREST(jsonOutput, doc){
+	try {
+		jsonParsed = JSON.parse(jsonOutput);
+		ref = jsonParsed['message']['items'][0];
+	} catch (e) {
+		Zotero.debug(e);
+		return false;
+	}
+	
+	if (jsonParsed['message']['total-results'] > 1) {
+		//Not sure what to do with multiple results 
+		//Should only return 1 item
+		throw new Error('Returned multiple results');
```

Comment:
This should be impossible for a DOI query, but if it happens, just parse the first one. Don't throw an error.